sources/reference/admiral-aws-marketplace.md

```diff
@@ -80,3 +80,28 @@ $ ssh -L 50003:1.2.3.4:50003 -C ubuntu@1.2.3.4
 - Click `Install`. This should set up Shippable services and bring up the UI.
   Once complete, you should be able to navigate to `http://1.2.3.4:50001` and
   log into Shippable using a Github account.
+
+
+### Setup superuser account
+
+Shippable UI also provides an admin panel which helps users to
+
+- monitor builds across the system
+- manage build nodes for either the system or individual subscriptions
+- manage user accounts across the system
+- view and manage any build, project and subscription
+- Perform other housekeeping tasks
+
+Only the accounts that have been added to superuser list will be able to access
```

Comment:
I would take out "the" after "only" and add it before "superuser list."  There is only one list of super users, but the accounts themselves aren't special otherwise.